### ISSUE TITLE
Installing on Windows

### ISSUE THREAD

**Issue by yuvalkry** [vagrawal/deepsphinx#4]
Hello,

  In the closing response to issue#3, it is written that deepsphinx was written for python 2.
  Tensorflow, however, requires python 3 for installing on Windows.

  Has anyone tried to install deepsphinx on windows?

  May I also add that the  backslashes in the Windows path may interfere
  with the input line structure.

   Thanks,
            Yuval

**Comment by vagrawal** [comment]
Thanks for reporting a very important bug. I will replace it with comma, and in that way I can parse it as csv. If you have any other suggestion, I will highly appreciate it.

In #3, I just meant that I mostly run this in python 2, as I use google cloud ML. I intend it to be fully compatible with both versions of python. If you find any difficulty in running with python 3, please report it here.

I will make it windows compatible in a few days, as I am busy tuning the model for better performance and releasing a pre-trained model.

**Comment by yuvalkry** [comment]
Hi,

About the input line, the transcription may contain a comma so it can be a problem
unlesss the transcription is enclosed between quotes (which also works for CSV).

The backslash in the path is not such a huge problem if you only take the first three fields
and treat the rest as a single field. You could also use a "|".

I agree that a csv-like solution, however, is the best :-)

The difficulty with Python 3 is bug #3 - if I understood correctly.

Could you please provide estimates of how much training data is required?
for example, because no dictionary is involved, does it require more data than pocketsphinx
to read a comparable performance? (not sure if worth opening another ticket for that).

Looking forward a windows-compatible version, thank you very much!

     Yuval

**Comment by vagrawal** [comment]
I fixed #3 with [5ed88ec](https://github.com/vagrawal/deepsphinx/commit/5ed88ecf98006dae1d35ffffc5d5880ccdc0b4b3), that's why I closed it.

I am using training data of around 80 hours and there is mild overfitting, but even after training for long, the validation accuracy does not decreases, for the (to be updated) default model, without any dropout.

**Comment by vagrawal** [comment]
I forgot to mention about escaping commas, but thought the exact same thing and use any of the available parsers for csv :-) .

I have designed the transcription in haste for temporary use, and stuck with it ever since. That's why I haven't used the more obvious way of doing it. Thanks again for noticing it.

**Comment by yuvalkry** [comment]
It works under Windows as well, great!

  Yuval
